packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v1-4-x/run-transaction-endpoint-v1.test.ts

```diff
@@ -141,8 +145,7 @@ test("runs tx on a Fabric v1.4.8 ledger", async (t: Test) => {
 
   {
     const req: RunTransactionRequest = {
-      keychainId,
-      keychainRef: keychainEntryKey,
+      fabricSigningCredential: fabricSigningCredential,
```

Comment:
@AzaharaC Same question as above, could this be simpler by just saying:

```typescript
fabricSigningCredential,
```

instead of 

```typescript
fabricSigningCredential: fabricSigningCredential,
```

Reply:
done